src/tags/bitwise.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @Author: RagingLink 
+ * @Date: 2020-08-17 14:03:17
+ * @Last Modified by: RagingLink
+ * @Last Modified time: 2021-05-20 23:03:10
+ *
+ * This project uses the AGPLv3 license. Please read the license file before using/adapting any of the code.
+ */
+
+ const Builder = require('../structures/TagBuilder');
+ let operators = {
```

Comment:
2 things here ; the variable can be declared as a constant **and** the naming bugs me. I know it's because it is not only operators in the object but rather like an `operatorsToOperations` dictionary. Do you think we can find a better name?